src/configuration/services/mysql.md

```diff
@@ -146,11 +146,10 @@ configuration:
 
 ## Access your MariaDB service
 
-Assuming your MariaDB relationship is named `database`, you can access it by connecting
-to a host named `database.internal` using the MySQL command line client.
+Assuming your MariaDB relationship is named `database`, the host name and port number obtained from `PLATFORM_RELATIONSHIPS` would be `database.internal` and `3306`. Open an [SSH session](/development/ssh.md) and run the MySQL command line client.
```

Comment:
FYI, it was using `SSH session` previously https://github.com/platformsh/platformsh-docs/pull/512/files/4b2e990617a3793dc5a192e9e4f057e0b1ba92a1#diff-fecddd292d23722cd2e396ab813d2a90L97